synapse/rest/media/v1/oembed.py

```diff
@@ -11,6 +11,7 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+import html as html5
```

Comment:
Is this to avoid conflicting with `html` below? I think I'd prefer to rename the variable instead of importing the module as something.